hdi/ARM-templates/UBsetup/unravel-AWS-vm-4611-setup.sh

```diff
@@ -0,0 +1,122 @@
+echo $RPMFILE > /tmp/rpmfilename
+curl -v -u  unravel:WillowRoad68 https://preview.unraveldata.com/unravel/staging-RPM/4.6.1/${RPMFILE} -o ${RPMFILE}
+
+BLOBSTORACCT=${1}
+BLOBPRIACKEY=${2}
+BLOBSECACKEY=${3}
+
+DLKSTOREACCT=${4}
+DLKCLIENTAID=${5}
+DLKCLIENTKEY=${6}
+DLKCLITOKEPT=${7}
+DLKCLIROPATH=${8}
+
+
+# Prepare the VM for unravel rpm install
+/usr/bin/yum install -y ntp
+/usr/bin/yum install -y libaio
+/usr/bin/yum install -y lzop
+/usr/bin/systemctl enable ntpd
+/usr/bin/systemctl start ntpd
+/usr/bin/systemctl disable firewalld
+/usr/bin/systemctl stop firewalld
+
+/usr/sbin/iptables -F
+
+/usr/sbin/setenforce 0
+/usr/bin/sed -i 's/enforcing/disabled/g' /etc/selinux/config /etc/selinux/config
+
+sleep 30
+
+
+# Prepare disk for unravel
+mkdir -p /srv
+
+DATADISK=`/usr/bin/lsblk |grep 500G | awk '{print $1}'`
+echo $DATADISK > /tmp/datadisk
+echo "/dev/${DATADISK}1" > /tmp/dataprap
+
+echo "Partitioning Disk ${DATADISK}"
+echo -e "o\nn\np\n1\n\n\nw" | fdisk /dev/${DATADISK}
+
+DATAPRAP=`cat /tmp/dataprap`
+DDISK=`cat /tmp/datadisk`
+/usr/sbin/mkfs -t ext4 ${DATAPRAP}
+
+DISKUUID=`/usr/sbin/blkid |grep ext4 |grep $DDISK  | awk '{ print $2}' |sed -e 's/"//g'`
+echo "${DISKUUID}    /srv   ext4 defaults  0 0" >> /etc/fstab
+
+/usr/bin/mount -a
+
+# install unravel rpm
+rpmfile_name=`cat /tmp/rpmfilename`
+/usr/bin/rpm  -U $rpmfile_name
+
+/usr/bin/sleep 5
+
+
+# Update Unravel Lic Key into the unravel.properties file
+# Obtain a valid unravel Lic Key file ; the following is just non working one
+echo "com.unraveldata.lic=1p6ed4s492012j5rb242rq3x3w702z1l455g501z2z4o2o4lo675555u3h" >> /usr/local/unravel/etc/unravel.properties
+echo "export CDH_CPATH="/usr/local/unravel/dlib/hdp2.6.x/*"" >> /usr/local/unravel/etc/unravel.ext.sh
+
+# Update Azure blob storage account credential in unravel.properties file
+# Update and uncomment the following lines to reflect your Azure blob storage account name and keys
+
+if [ $BLOBSTORACCT != "NONE" ] && [ $BLOBPRIACKEY != "NONE" ] && [ $BLOBSECACKEY != "NONE" ]; then
+
+   echo "blob storage account name is ${BLOBSTORACCT}"
+   echo "blob primary access key is ${BLOBPRIACKEY}"
+   echo "blob secondary access key is ${BLOBSECACKEY}"
+   echo "# Adding Blob Storage Account information, Update and uncomment following lines" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.storage-account-name-1=fs.azure.account.key.${BLOBSTORACCT}.blob.core.windows.net" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.primary-access-key=${BLOBPRIACKEY}" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.storage-account-name-2=fs.azure.account.key.${BLOBSTORACCT}.blob.core.windows.net" >> /usr/local/unravel/etc/unravel.properties
+   echo "com.unraveldata.hdinsight.secondary-access-key=${BLOBSECACKEY}" >> /usr/local/unravel/etc/unravel.properties
```

Comment:
The properties for WASB has changed please use the new property name from the link:
`Step 5.C` Properties for WASB Storage (Unravel 4.5.2.x)
https://docs.unraveldata.com/en/azure-hdi-part1-vm.html#al_idp101525